skyportal/handlers/api/candidate.py

```diff
@@ -500,11 +558,15 @@ def patch(self, obj_id):
 
 
 def grab_query_results_page(q, total_matches, page, n_items_per_page, items_name):
+    # The query can return multiple rows for candidates with multiple annotations
+    # Getting the query results and doing the limit/offset on the Python side
+    # simplifies the deduplication
+    items = q.all()
```

Comment:
See comment below -- this could be a huge number of candidates. I think we want to keep the `limit`